system_launch/launch/system.launch.xml

```diff
@@ -2,21 +2,22 @@
 <launch>
   <arg name="run_mode" default="online" doc="options: online, planning_simulation"/>
 
-  <group ns="/system">
+  <group>
     <!-- State Monitor -->
-    <include file="$(find autoware_state_monitor)/launch/autoware_state_monitor.launch">
-      <arg name="config_file" if="$(eval run_mode == 'online')" value="$(find autoware_state_monitor)/config/autoware_state_monitor.yaml" />
-      <arg name="config_file" if="$(eval run_mode == 'planning_simulation')" value="$(find autoware_state_monitor)/config/autoware_state_monitor.planning_simulation.yaml" />
-    </include>
+    <push-ros-namespace namespace="/system"/>
+    <!-- <include file="$(find-pkg-share autoware_state_monitor)/launch/autoware_state_monitor.launch.xml">
+      <arg name="config_file" if="$(eval run_mode == 'online')" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.yaml" />
+      <arg name="config_file" if="$(eval run_mode == 'planning_simulation')" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.planning_simulation.yaml" />
+    </include> -->
```

Comment:
Any reason you commended out here?

This is going to be ported like below.

```
    <push-ros-namespace namespace="/system"/>
    <include file="$(find-pkg-share autoware_state_monitor)/launch/autoware_state_monitor.launch.xml">
      <arg name="config_file" if="$(eval &quot;'$(var run_mode)'=='online'&quot;)" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.yaml" />
      <arg name="config_file" if="$(eval &quot;'$(var run_mode)'=='planning_simulation'&quot;)" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.planning_simulation.yaml" />
    </include>
```

Reply:
When I started this PR the `autoware_state_monitor` package hadn't been ported yet. Re-added in 624243f63221ceea313784cfeed5a27f282a6801